news/_posts/2023-05-11-ro-crate.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "BY-COVID and RO-Crate collaboration brings new topic: FAIR Data, Workflows & More"
+tags: [new topic,new feature]
+contributions:
+  authorship: [hexylena,simleo,pauldg,stain]
+  funding: [by-covid]
+layout: news
+abbreviations:
+  FAIR: Findable Accessible Interoperable Reusable
+---
+
+As part of the BY-COVID project, the RO-Crate team and BY-COVID members have
+brought several RO-Crate related tutorials into the Galaxy Training Network, under a brand new {FAIR} topic!
+
+## What are RO-Crates?
+
+> RO-Crate is a community effort to establish a lightweight approach to packaging research data with their metadata. It is based on schema.org annotations in JSON-LD, and aims to make best-practice in formal metadata description accessible and practical for use in a wider variety of situations, from an individual researcher working with a folder of data, to large data-intensive computational research environments. -- [https://www.researchobject.org/ro-crate/](https://www.researchobject.org/ro-crate/)
+
+These are *the* way to package up your research in a well annotated and FAIR manner.
+
+There are a number of ["profiles" within RO-Crates](https://www.researchobject.org/ro-crate/profiles.html), we are currently focused on the "Workflow Run" profile but we encourage you to look into the [other profiles](https://www.researchobject.org/ro-crate/profiles.html)
+
+## What's new for you?
+
+As part of this collaboration we bring you several new tutorials:
+
+- ["Workflow Run RO-Crate Introduction"]({% link topics/fair/tutorials/ro-crate-workflow-run-ro-crate/tutorial.md %}), an introduction to Workflow Run profile RO-Crates
```

Comment:
"an introduction to Workflow Run profile RO-Crates" :arrow_right: "an introduction to RO-Crate profiles to represent workflow executions"